testflight_invite.py

```diff
@@ -150,33 +198,35 @@ def numTesters(self):
 
     def addTester(self, email, firstname='', lastname=''):
         self.login()
+        params = { 'email': email, 'firstName': firstname, 'lastName': lastname }
+        try:
+            text = self.readData(
+                'https://itunesconnect.apple.com/testflight/v2/providers/{teamId}/apps/{appId}/testers'.format(
+                    teamId=self.contentProviderId, appId=self.appId
+                ),
+                json.dumps(params),
+                headers={'Content-Type': 'application/json'}
+            )
+        except urllib2.HTTPError as e:
+            raise
```

Comment:
If we are just going to raise it, let's not even capture try `try/catch` and rely on the calling function to handle error logic.